action.yml

```diff
@@ -6,6 +6,11 @@ inputs:
       type: string
       default: nrfconnect/sdk-nrf
       description: 'target repository for manifest PR'
+    base-branch:
+      required: false
+      type: string
+      default: main
+      description: 'base/target branch for manifest PR'
```

Comment:
I believe this should be used after line 78 (in checkout phase).

Reply:
added:  https://github.com/actions/checkout/blob/main/README.md?plain=1#L30